### DISCUSSION TITLE
Passkey

### DISCUSSION THREAD

**Discussion by Scottwint** [ente-io/ente#2204] [General]
If I activate passkey on ente Auth, where will the passkey be stored? Is it on ente Auth? If yes, does the passkey synchronize in case I change my phone or lose?

**Comment by Vantesh** [comment]
Passkeys will be stored on your default password manager depending on the device you are on and about synchronizing, it depends with the password manager, most do sync the passkeys so you don't have to worry about changing your device

**Reply by PackElend** [reply]
Any chance that Ente will ever be able to store passkeys to allow synchronization between Paltrofam and devices?
